index.js

```diff
@@ -2,6 +2,7 @@ var fs = require('fs');
 var path = require('path');
 var join = path.join;
 var mkdirp = require('mkdirp');
+var pathIsAbsolute = require('path-is-absolute');
```

Comment:
Yeah, this is a debate topic:
+ why small npm modules eco system is good: https://github.com/sindresorhus/ama/issues/10#issuecomment-117766328
-  history with leftpad: http://www.haneycodes.net/npm-left-pad-have-we-forgotten-how-to-program/

Can inline that code inside project, do I need?